src/lib/watcher/runner.ts

```diff
@@ -126,25 +119,51 @@ register({
     IPC.client.senders.moduleImported({ filePath })
   })
 
-  if (!process.env.NEXUS_EVAL) {
-    throw new Error('process.env.NEXUS_EVAL is required')
-  }
+  const startModuleFileName = layout.sourceRoot + '/index.ts'
+  const startModule = tsNodeRegister.compile(
+    createStartModuleContent({
```

Comment:
Side note (we talked about it in the past already): I see how it makes things simpler to not pass the start module from the dev command down to the runner, but I'm concerned about all the stuff we put here. We're tightly coupling the watcher with a lot of internal stuff from everywhere.

Initially, this runner was just a generic code to run some code and report errors to the parent.

On the one hand it reduces complexity, which is great, but it makes the abstraction bad. Ideally, everything related to our internals should go away, including the context type extraction

Reply:
Here you, I've improved the situation from before by having the path to the runner be passed down as as argument.

I suppose we may want the watcher to own the runner. But compared to how this PR had things, this seems better now:

![image](https://user-images.githubusercontent.com/284476/77647721-4f290d00-6f3d-11ea-9462-9748a123f22b.png)

Reply:
Hm, will deal with this in a follow up PR... I broke things 🙈